packages/react/src/patterns/sub-patterns/Layout/__stories__/Layout.stories.js

```diff
@@ -78,78 +85,124 @@ storiesOf('Patterns (Sub-Patterns)|Layout', module)
-              volutpat ac tincidunt vitae semper quis lectus nulla.
-            </p>
-
-            <p style={{ paddingBottom: '1rem' }}>
-              Eu facilisis sed odio morbi quis commodo. Non tellus orci ac
-              auctor augue mauris augue. Accumsan tortor posuere ac ut. Orci
-              sagittis eu volutpat odio facilisis. Vestibulum mattis ullamcorper
-              velit sed ullamcorper morbi. Vestibulum rhoncus est pellentesque
-              elit ullamcorper dignissim cras tincidunt. Id leo in vitae turpis
-              massa sed elementum. Euismod elementum nisi quis eleifend quam
-              adipiscing vitae proin sagittis. Duis ultricies lacus sed turpis
-              tincidunt id aliquet. Nibh mauris cursus mattis molestie a iaculis
-              at erat pellentesque. Tempus iaculis urna id volutpat lacus.
-              Rhoncus aenean vel elit scelerisque mauris.
-            </p>
-
-            <p style={{ paddingBottom: '1rem' }}>
-              Hac habitasse platea dictumst quisque sagittis. Faucibus nisl
-              tincidunt eget nullam non nisi est sit. Volutpat ac tincidunt
-              vitae semper quis lectus. Nisi lacus sed viverra tellus in hac.
-              Rhoncus dolor purus non enim. Suspendisse sed nisi lacus sed
-              viverra. Faucibus vitae aliquet nec ullamcorper sit amet risus
-              nullam eget. Augue interdum velit euismod in pellentesque massa.
-              Fermentum iaculis eu non diam phasellus vestibulum lorem sed.
-              Volutpat maecenas volutpat blandit aliquam etiam erat velit
-              scelerisque.
-            </p>
-
-            <p style={{ paddingBottom: '1rem' }}>
-              Mi in nulla posuere sollicitudin aliquam ultrices sagittis. Elit
-              pellentesque habitant morbi tristique senectus. Nunc non blandit
-              massa enim nec dui nunc mattis enim. In egestas erat imperdiet sed
-              euismod nisi porta lorem mollis. Donec pretium vulputate sapien
-              nec sagittis aliquam malesuada bibendum arcu. Auctor urna nunc id
-              cursus metus aliquam eleifend mi. Velit aliquet sagittis id
-              consectetur. Arcu cursus vitae congue mauris. Gravida rutrum
-              quisque non tellus orci ac. At erat pellentesque adipiscing
-              commodo elit at. Amet tellus cras adipiscing enim eu turpis
-              egestas. Purus gravida quis blandit turpis cursus in hac
-              habitasse. Interdum velit laoreet id donec ultrices tincidunt arcu
-              non sodales. Interdum posuere lorem ipsum dolor sit. Quam quisque
-              id diam vel quam elementum pulvinar etiam non. Massa enim nec dui
-              nunc mattis enim. Ac tortor vitae purus faucibus ornare
-              suspendisse sed. Non pulvinar neque laoreet suspendisse.
-            </p>
+
+          <div className="bx--grid">
+            <div className="bx--row">
+              <div className="bx--col">
```

Comment:
This is already included with the parent `Layout` above. Not needed for demonstration purposes.